src/site/_includes/layouts/_base.njk

```diff
@@ -0,0 +1,46 @@
+{% from "moduk/components/header/macro.njk" import modukHeader %}
```

Comment:
Refactored the templates a bit (so the main part of the body in the base template is empty).